package-lock.json

```diff
@@ -52,12 +52,13 @@
       }
```

Comment:
I wouldn't expect this needed to be updated unless you are explicitly using functionality from newer packages, so I would probably revert to main unless there is something specific you need to update for.